docs/website/data/integrations.yaml

```diff
@@ -510,6 +510,23 @@ integrations:
       - styra
       - build.security
 
+  springsecurity-reactive:
+    title: Authorization for Java Spring Security Reactive
+    description: This integration provides a simple implementation of a ReactiveAuthorizationManager<ServerHttpRequest> for Spring Security Reactive that uses OPA for making API authorization decisions. In addition, the library provides out-of-the-box token-based JWT Authentication, allowing transparent RBAC implementations using JSON Web Tokens.
+    labels:
+      layer: network
+      category: application
+    software:
+      - javaspringsecurity
+    code:
+      - https://github.com/massenz/jwt-opa
+    tutorials:
+      - https://github.com/massenz/jwt-opa/blob/main/webapp-example/src/main/java/com/alertavert/opademo/JwtDemoApplication.java
```

Comment:
A java source file isn't really a tutorial, is it? :)

Reply:
LoL - no kidding.
But that is an example web app and shows how to integrate the `jwt-opa` library.
Fixed.